airflow/__init__.py

```diff
@@ -37,9 +37,6 @@
 from airflow.models import DAG
 from airflow.exceptions import AirflowException
 
-if settings.DAGS_FOLDER not in sys.path:
```

Comment:
Why are we removing this from here??

I see that we have the same logic in `airflow/settings.py` but have not investigated why!

Reply:
For me, appending to `sys.path` is a code smell imo, I think it is good to just remove it :-)

Reply:
From the commit message:

> There was an import cycle form settings -> logging_config ->
> module_loading -> settings that needed to be broken on Python 2 - so I
> have moved all adjusting of sys.path in to the settings module

So not removed from code base, but isloated to a single place at least.